lib/trento/sap_systems/services/health_summary_service.ex

```diff
@@ -27,6 +27,7 @@ defmodule Trento.SapSystems.Services.HealthSummaryService do
     |> where([s], is_nil(s.deregistered_at))
     |> order_by(asc: :sid)
     |> Repo.all()
+    |> Repo.preload(:database)
```

Comment:
Remove this `preload`, it is already preloaded in line 33 (or remove the next one, just leave one of them)

Reply:
done, rebase magic!